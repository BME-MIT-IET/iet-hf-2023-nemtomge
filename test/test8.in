loadmap map.json 
start V1 
set Gencode LA1 Stun 30 30 
set Scientist V1 LA1 
give Gencode V1 Stun 30 30
show Gencode V1
touch V1
1
show Gencode V1
exit
