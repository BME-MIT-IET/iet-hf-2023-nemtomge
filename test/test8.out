Map from file map.json loaded
The game has been started. Players are: V1 
Genetic code Stun aminoacid: 30 nucleotide: 30 has been placed on field LA1 
Scientist V1 set on field LA1 
Scientist V1 has been given gencode Stun aminoacid: 30 nucleotide: 30
V1's genetic codes are:
Stun(3)
Choose one to pick: 
0 - cancel
1 - Stun(3)
V1's genetic codes are:
Stun(3)
